File: Q13.cpp
#include <iostream>

using namespace std;

int main()
{
    int numero_conta;
    float saldo, debito, credito, saldo_atual;
    cout << "Digite o número da conta do cliente: ";
    cin >> numero_conta;
    cout << "Digite o saldo do cliente: ";
    cin >> saldo;
    cout << "Digite o débito do cliente: ";
    cin >> debito;
    cout << "Digite o crédito do cliente: ";
    cin >> credito;
    saldo_atual = saldo - debito + credito;
    if (saldo_atual >= 0)
    {
        cout << "Saldo Positivo." << endl;
    }
    else
    {
        cout << "Saldo Negativo." << endl;
    }
    return 0;
}

File: Q13.java
import java.util.Scanner;

public class Q13 
{
    public static Scanner leia = new Scanner(System.in);

    public static void main(String args[]) 
    {
        int numero_conta;
        float saldo, debito, credito, saldo_atual;
        System.out.print("Digite o número da conta do cliente: ");
        numero_conta = leia.nextInt();
        System.out.print("Digite o saldo do cliente: ");
        saldo = leia.nextFloat();
        System.out.print("Digite o débito do cliente: ");
        debito = leia.nextFloat();
        System.out.print("Digite o crédito do cliente: ");
        credito = leia.nextFloat();
        saldo_atual = saldo - debito + credito;
        if (saldo_atual >= 0) 
        {
            System.out.println("Saldo Positivo.");
        } 
        else 
        {
            System.out.println("Saldo Negativo.");
        }
    }
}